src/sentry/search/snuba/executors.py

```diff
@@ -213,6 +232,8 @@ def query(
         search_filters,
         date_from,
         date_to,
+        *args,
+        **kwargs
```

Comment:
Why do you need this? The method defined in the parent class does not have optional parameters.